interface/super/manage_site_files.php

```diff
@@ -99,6 +100,42 @@
   }
 
 }
+
+/**
+ * Thumbnails generator
+ * generating  thumbnail image to all images files from documents table
+ */
+
+switch((int)$GLOBALS['document_storage_method']){
+    case 0:
+        $storage_method = 'on Disc';
+        break;
+    case 1:
+        $storage_method = 'in CouchDB';
+        break;
+}
```

Comment:
above block is not needed and not good from internationalization standpoint. Simply use the $GLOBALS['document_storage_method'] below to decide whether to state:
The thumbnails will be stored on Hard Disk
The thumbnails will be stored in CouchDB